ci/testnet-automation.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
```

Comment:
`cd "$(dirname "$0")/.."` for robustness (enables this script to be run from anywhere and still work)